src/parse/pdxfile.rs

```diff
@@ -69,6 +96,75 @@ impl Calculation {
             | Calculation::Divide(_) => false,
         }
     }
+}
+
+/// Keeps the stack of calculations and current, top of the stack for processing.
+#[derive(Debug)]
+struct Calculator {
+    stack: Vec<Vec<Calculation>>,
+    current: Vec<Calculation>,
+}
+
+impl Calculator {
+    fn new() -> Self {
+        Self { stack: Vec::new(), current: Vec::new() }
+    }
+
+    fn start(&mut self) {
+        self.stack.clear();
+        self.current.clear();
+    }
+
+    /// Register a part of a calculation, either an operator or a [`Value`](`Calculation::Value`).
```

Comment:
I think this comment was never true :) From the body it's clear that the `op` must be an operator, not a `Value`.

Reply:
Right now, it does not forbid taking a Value variant. Should there be a check for it do you think?

Reply:
I think it's OK to just fix the comment.

I realize it was my comment and not yours but it would be convenient to fix it here :)